examples/Assistants/Example04_AllTheTools.cs

```diff
@@ -171,8 +172,8 @@ PageableCollection<ThreadMessage> messages
             #endregion
 
             #region List run steps for details about tool calls
-            PageableCollection<RunStep> runSteps = client.GetRunSteps(run, resultOrder: ListOrder.OldestFirst);
-            foreach (RunStep step in runSteps)
+            PageCollection<RunStep> runSteps = client.GetRunSteps(run, new() { Order = ListOrder.OldestFirst });
```

Comment:
Ignorable/opinion-only: even though it's more verbose, I like either using the explicit constructor named method parameter in example code like this; until you've learned the patterns, it's to infer what's being `new()`'d.

Reply:
Yep, happy to. I was optimizing for characters, but if we feel this isn't a familiar pattern yet, I'm happy to go with what we think will be more helpful to users.